markup5ever/util/buffer_queue.rs

```diff
@@ -105,13 +102,13 @@ impl BufferQueue {
     /// Get the next character if one is available, removing it from the queue.
     ///
     /// This function manages the buffers, removing them as they become empty.
-    pub fn next(&mut self) -> Option<char> {
+    pub fn next_char(&mut self) -> Option<char> {
```

Comment:
clippy complained, thinks it could be confused with `std::iter::Iterator::next`,
also `pub` so if we need a version bump anyways, we could update this as well,
or we can revert